projects/br_scs/src/Mirza/BusinessRegistry/API.hs

```diff
@@ -74,13 +74,10 @@ type PublicAPI =
 
 
 type PrivateAPI =
-       -- TODO: Note the following end point does't really make sense any more, it has only been left in place for now
-       --       because the plan is to basically refactor it into an associate user with a business end point.
-       "user"      :> "add"      :> ReqBody '[JSON] NewUser     :> Post '[JSON] UserId
-  :<|> "business"  :> "add"      :> ReqBody '[JSON] NewBusiness :> Post '[JSON] GS1CompanyPrefix
-  :<|> "key"       :> "add"      :> ReqBody '[JSON] JWK         :> QueryParam "expirationTime" ExpirationTime :> Post '[JSON] BRKeyId
-  :<|> "key"       :> "revoke"   :> Capture "keyId" BRKeyId     :> Post '[JSON] RevocationTime
-  :<|> "location"  :> "add"      :> ReqBody '[JSON] NewLocation :> Post '[JSON] LocationId
-  :<|> "company"   :> Get '[JSON] [BusinessResponse]
-
-
+       "user"          :> "add"      :> Put '[JSON] NoContent
+  :<|> "organisation"  :> "add"      :> ReqBody '[JSON] NewBusiness             :> Post '[JSON] GS1CompanyPrefix
+  :<|> "organisation"  :> Capture "gs1CompanyPrefix" GS1CompanyPrefix :> "user" :> Capture "userId" UserId                    :> Put '[JSON] NoContent
+  :<|> "key"           :> "add"      :> ReqBody '[JSON] JWK                     :> QueryParam "expirationTime" ExpirationTime :> Post '[JSON] BRKeyId
+  :<|> "key"           :> "revoke"   :> Capture "keyId" BRKeyId                 :> Post '[JSON] RevocationTime
+  :<|> "location"      :> "add"      :> ReqBody '[JSON] NewLocation             :> Post '[JSON] LocationId
+  :<|> "user"          :> "listorganisations"                                   :> Get '[JSON] [BusinessResponse]
```

Comment:
On the same vein, maybe make this `/list/user/organisation`?

Reply:
https://github.com/data61/Mirza/pull/430#discussion_r281877446